docs/src/main/sphinx/release/release-407.md

```diff
@@ -0,0 +1,44 @@
+# Release 407 (1 Feb 2023)
+
+## General
+
+* Add support for adaptive task request sizing. It can be enabled with the
+  `query.remote-task.enable-adaptive-request-size` configuration property and
+  configured with the `query.remote-task.max-request-size`,
+  `query.remote-task.request-size-headroom`, and
+  `query.remote-task.guaranteed-splits-per-task` configuration properties or
+  their respective session properties. ({issue}`15721`)
+* Improve query performance when a predicate evaluates to a null value. ({issue}`15744`)
+* Improve performance of queries with joins on the output of global
+  aggregations. ({issue}`15858`)
+* Fix precision loss when converting `time` values with a precision higher than
+  three and `time with time zone` values with lower precision. ({issue}`15861`)
+* Change the `table_type` of materialized views in `information_schema.tables`
+  from `BASE TABLE` to `MATERIALIZED VIEW`. ({issue}`15350`)
+
+## Delta Lake connector
+
+* Remove support for the legacy, non-optimized Parquet writer. ({issue}`15436`)
+* Improve performance of reading decimal columns from Parquet files. ({issue}`15713`)
+* Improve performance of reading numeric types from Parquet files. ({issue}`15850`)
+
+## Hive connector
+
+* Improve performance of reading decimal columns from Parquet files. ({issue}`15713`)
+* Improve performance of reading numeric types from Parquet files. ({issue}`15850`)
+* Reduce query latency. ({issue}`15811`)
```

Comment:
I think #15811 reduces query latency in Delta Lake connector as well. 
cc: @sopel39 @findepi

Reply:
I don't think it does. Delta connector doesn't read stats from the metastore.